### PR TITLE
feat(docs): server stub generation

### PR DESCRIPTION
**Pull request by dannysheridan**

### CODE REVIEW

File: fern/pages/servers/express.mdx [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+{
+  "error": "MovieDoesNotExistError",
+  "errorInstanceId": "f6e1d69c-bf97-42d5-bc89-5e42773e3880",
+  "content": "oceans-11"
+}
+```
+
+</Steps>
+
+## Advanced Configurations
```
Comment (dannysheridan) [inline fern/pages/servers/express.mdx:105]: @zachkirsch good ol' server stubs coming back into play

Comment (zachkirsch) [reply fern/pages/servers/express.mdx:105]: lol love it. I think I have “server INTERFACES not stubs” still burned into my memory

File: fern/pages/servers/express.mdx [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+<Note>Make sure to enable `allowSyntheticDefaultImports` in your `tsconfig.json` when using this generator.</Note>
+
+### Implement the functions 
+
+Provide the business logic by implementing each function. For example, if we were implenting the IMDb API, we would implement the `get_movie` function.
+
+<CodeBlock title="src/movies_service.py">
+```python
+from .generated.fern import AbstractMoviesService, Movie, MovieDoesNotExistError, MovieId
```
Comment (zachkirsch) [inline fern/pages/servers/express.mdx:65]: While I’m here 🫣 this file should prob have express examples not fastapi examples

**Comment by dannysheridan** [comment]
This is stale. If we want to document it in the future, we should start from scratch and do one at a time.

**Pull request closed without merging**